java-symbolic-execution/java-symbolic-execution-plugin/src/test/java/org/sonar/java/se/checks/UnclosedResourcesCheckTest.java

```diff
@@ -34,12 +34,22 @@ void test() {
   }
 
   @Test
-  void test_lombok() {
+  void doesNotRaiseOnLombokCleanupAnnotatedVariable() {
     SECheckVerifier.newVerifier()
       .onFile("src/test/files/se/UnclosedResourcesLombokCheck.java")
       .withCheck(new UnclosedResourcesCheck())
       .withClassPath(SETestUtils.CLASS_PATH)
-      .verifyIssues();
+      .verifyNoIssues();
+  }
+
+  @Test
+  void doesNotRaiseOnLombokCleanupAnnotatedVariable_noSemantic() {
```

Comment:
If we are not consistent within the file, we could still try to be consistent in this signature.
```suggestion
  void doesNotRaiseOnLombokCleanupAnnotatedVariableNoSemantic() {
```